packages/web/server/queries/indexer/pool-aprs.ts

```diff
@@ -0,0 +1,18 @@
+import { apiClient } from "@osmosis-labs/utils";
+
+import { NUMIA_INDEXER_BASEURL } from ".";
+
+type PoolApr = {
+  pool_id: string;
+  swap_fees: number;
+  superfluid: number;
+  osmosis: number;
+  boost: number;
+  total_apr: number;
+};
+
+/** Queries numia for a breakdown of APRs per pool. */
+export function queryPoolAprs(): Promise<PoolApr[]> {
+  const url = new URL("/pools_apr", NUMIA_INDEXER_BASEURL);
+  return apiClient(url.toString());
+}
```

Comment:
I believe we already have a query like this within the Numia namespace in queries, following Staking.